internal/v2/remote_signer/remote_signer_test.go

```diff
@@ -120,6 +119,22 @@ func TestSign(t *testing.T) {
 	if !bytes.Equal(gotSignedBytes, wantSignedBytes) {
 		t.Errorf("gotSignedBytes = %v, wantSignedBytes = %v", gotSignedBytes, wantSignedBytes)
 	}
+	if err = rsa.VerifyPKCS1v15(clientx509Cert.PublicKey.(*rsa.PublicKey), crypto.SHA256, hsha256[:], gotSignedBytes); err != nil {
+		t.Errorf("failed to verify RSA PKCS #1 v1.5 signature: %v", err)
+	}
+
+	// Test RSA PSS signature algorithm.
+	s = New(clientx509Cert, cstream, &commonpbv1.Identity {
+		IdentityOneof: &commonpbv1.Identity_Hostname {
+			Hostname: "client_hostname",
+		},
+	} , s2av2pb.SignatureAlgorithm_S2A_SSL_SIGN_RSA_PSS_RSAE_SHA256)
+	pssSignerOpts := rsa.PSSOptions {-1, crypto.SHA256 }
```

Comment:
readability nit: same here, I think it should be `pssSignerOpts := &rsa.PSSOptions{SaltLength: -1, Hash: crypto.SHA256}`?